oauth/authorization.md

```diff
@@ -15,78 +15,290 @@ Here are links to lessons that should be completed before this lesson:
 - [Adding Auth0 to your Project](/oauth/o-auth.md) - Especially the 'Authentication v. Authorization' slides
 - [Node.js](/node-js/node-js.md)
 - [Express.js](/express-js/express.md)
+
+
 ### Motivation
 
-Here's why this topic should be learned. Strengthen this section by including a company that uses this skillset, and a screenshot example if possible.
+Many app developers need a way to persist user information from session to session, but they also want to provide a smooth experience for those users. The OAuth protocol allows user to "log in" to an app using their Google account (or other social media accounts). This removes friction for the user because they don't need to create a new profile or remember a new password. OAuth can be used for other types of integration, like allowing users to give an app permission to access the user's bank balances or Dropbox files. This is beyond the scope of this lesson but may be valuable to you in the future.
 
 ### Objectives
 
 **Participants will be able to:**
 
-- leave this lesson knowing this
-- and how to do this on their own
-- Even more things
-- Even more things
+- add authentication and authorization to an Express.js API.
 
 ### Specific Things To Teach
 
-- Focus on this specific talking point
-- and this
-	- This is a sub-thing about the thing
-- Even more things about the things
-- Even more things about the things
+- Configure Auth0 APIs
+- Validate Access Tokens
+- Protect API Endpoints
 
-### Materials
+### Supplement Materials
 
 - [Authorization Slides](https://docs.google.com/presentation/d/1p7fa20o7lRyvXuXhsFz8MfHktY5MaqSPSrLdNjocNcE/edit?usp=sharing)
-- [Example video (10 min)](https://example.com) - Write a very short description of this resource.
-- [Other example article(20 min read)](https://otherexample.com) - Answer the 3 questions at the end.
-(Be sure to explain how to use these materials in directions in one of the sections below (ie: lesson, guided practice).  If you are not assigning them somehow, they should be placed under Supplemental Materials at the bottom.) 
+- [Example video (10 min)](https://example.com)
 
 ### Lesson
 
 Here's text about introducing something and how it works.
```

Comment:
Remove default text

Reply:
This is unchanged because still there is no video for lesson.